notebooks/GWAS_blood_pressure_p2/main.py

```diff
@@ -476,14 +502,14 @@
     Check that these files were successfully loaded to the bucket:
     """
 with herzog.Cell("python"):
-    get_ipython().system(" gsutil ls {bucket + 'MyProject_MAFgt0.01.vcf.bgz/*'}")  # noqa: F821
+    get_ipython().system(" gsutil ls {bucket + 'MyProject_MAFgt0.01.vcf.bgz/*'}")
```

Comment:
this might still need a F821

Reply:
Let's ignore this for now. I'll fix it in a future PR.

Reply:
Turns out it passes without issue so a fix isn't needed.